### DISCUSSION TITLE
I keep getting an error when trying to download

### DISCUSSION THREAD

**Discussion by hexactement** [aclap-dev/video-downloadhelper#342] [Help and Q&A]
### Your question

Hi,
After 1% I have the message: no media to process. With all the videos.
Message :
moz-extension://da6f6df5-cbaf-4057-ae2e-e83034f87359/background/main.js:1
receive@moz-extension://da6f6df5-cbaf-4057-ae2e-e83034f87359/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://da6f6df5-cbaf-4057-ae2e-e83034f87359/background/main.js:1:42626

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: fr
license: accepted
key (only 16 first characters): 774305af97cc4696
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\franc",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Is that on a particular site or across all sites ? Have you verified the companion app is not blocked by anti virus & firewall.

**Reply by hexactement** [reply]
Thank you for your reply. 
This does it on all sites. and I tested by deactivating the antivirus and changing the destination folder. 
It's the same.

**Reply by Mattmjs** [reply]
And is the firewall built into the anti virus or separate ? Did you check the firewall ?
